### DISCUSSION TITLE
Missing Gridpoint

### DISCUSSION THREAD

**Discussion by richhersey** [weather-gov/api#707] [Q&A] [answered]
I'm trying to find the gridpoint for this location near Kenai Fjords National Park. When I enter Kenai Fjords National Park into forecast.weather.gov it gives me these lat long coordinates, which I placed into the API to produce this;

https://api.weather.gov/points/59.8157,-150.1086

When I access that API there is not a gridpoint. Here are the results;

 "gridId": null,
 "gridX": null,
 "gridY": null,
 
 If I do the same search for Katmai National Park and Preserve, Alaska, I can see the API, https://api.weather.gov/points/58.5626,-154.9788, produces the gridpoint data result;

"gridId": "ALU",
"gridX": 763,
"gridY": 201,

Is there an error within https://api.weather.gov/points/59.8157,-150.1086 or am I doing something wrong? 

Cheers!
Rich

**Comment by sullynole** [comment, marked as answer]
This is a known issue within the recently updated Anchorage zones.  It's mentioned on the Updates tab here: https://www.weather.gov/documentation/services-web-api
